src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/PoisonUsage.txt

```diff
@@ -1,551 +1,635 @@
 <PrebuiltLeakReport>
-  <File Path="dotnet-sdk-x.y.z/sdk/x.y.z/Roslyn/bincore/runtimes/win/lib/netcoreapp3.1/System.Text.Encoding.CodePages.dll">
+  <File Path="dotnet-apphost-pack-x.y.z.tar/packs/Microsoft.NETCore.App.Host/x.y.z/runtimes/fedora.33-x64/native/coreclr_delegates.h">
```

Comment:
The baselines should not contain any rids.  This would tie the baseline to a particular rid and always fail if run on any other rid.  The intend is that https://github.com/dotnet/installer/blob/main/src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs#L95 removes these.

Reply:
Thanks for the catch, I must have missed the RIDs if there were any in the old one.

Reply:
I just wanted to check to ensure sure the test code that was intended to handle these was working correctly.  If you grabbed the baseline from the test, I'm concerned something might be wrong.

Reply:
BTW, The edits you made look correct and what I would expect.